### PR TITLE
added suggesion 1614

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

added suggesion [1614](https://canary.discord.com/channels/565557184348422174/627048923122499584/887054754092580936)
## Proposed changes

Added the ability for the nethergoldpan to drop soul soil from soulsand
This pr is however blocked by #3646

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.14.* - 1.19.*).
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/NetherGoldPan.java [hunk 1]
```diff
@@ -41,6 +41,7 @@ public NetherGoldPan(ItemGroup itemGroup, SlimefunItemStack item, RecipeType rec
         settings.add(new GoldPanDrop(this, "chance.GOLD_NUGGET", 25, new ItemStack(Material.GOLD_NUGGET)));
         settings.add(new GoldPanDrop(this, "chance.NETHER_WART", 10, new ItemStack(Material.NETHER_WART)));
         settings.add(new GoldPanDrop(this, "chance.BLAZE_POWDER", 8, new ItemStack(Material.BLAZE_POWDER)));
+        settings.add(new GoldPanDrop(this, "change.SOUL_SOIL", 5, new ItemStack(Material.SOUL_SOIL)));
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/NetherGoldPan.java:44]: *coughs*
```suggestion
        settings.add(new GoldPanDrop(this, "chance.SOUL_SOIL", 5, new ItemStack(Material.SOUL_SOIL)));
```

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Comment by TheBusyBiscuit** [comment]
Also I am not particularly a fan of the idea of "panning" soul soil from soul sand...

**Comment by J3fftw1** [comment]
> Also I am not particularly a fan of the idea of "panning" soul soil from soul sand...

Yea its a bit weird it was accepted however tho. Any suggestions of what machine this would fit tho

**Comment by Sefiraat** [comment]
It's still a bit loose, but the composter? While not great I'd say more fitting than panning

**Comment by J3fftw1** [comment]
> uhm, [the PR](https://github.com/Slimefun/Slimefun4/pull/3269) I still have open adds soul soil as an input material this wont really work well together

Yea that wont work. Yours makes more sense aswell. Closing this. Someone should really dig out #approved channel its such a mess in there with stuff already accepted and stuff that doesnt make sense anymore

**Pull request closed without merging**